Doc/library/time.rst

```diff
@@ -299,6 +299,11 @@ Functions
    Use :func:`monotonic_ns` to avoid the precision loss caused by the
    :class:`float` type.
 
+   .. impl-detail::
+
+      On CPython, use the same clock as :func:`time.perf_counter` and is a
+      monotonic clock, i.e. a clock that cannot go backwards.
```

Comment:
This makes more sense under _perf_counter_ were it was. All _monotonic_ function should always be monotonic, by definition.